Packages/Web/lib/util.flp

```diff
@@ -471,8 +471,12 @@ func trackOffsets(_ times, _ duration)
     return offsets
 end
 
-func trackValues(property, _ keyvalues, layer, imageFolderPath, importImage, _ duration)
+func trackValues(property, _ keyvalues, timeline, layer, imageFolderPath, importImage, _ duration)
     var values = []
+    if property.id == "transform.rotation.z"
+        return transformValues(keyvalues, timeline, layer)
+    end
+
     for keyvalue in keyvalues
         var value = cssValue(property: property, value: keyvalue.value, layer: layer, imageFolderPath: imageFolderPath, importImage: importImage)
```

Comment:
lets pass the timeline into `cssValue` function and let it return the appropriate transform value.